### DISCUSSION TITLE
Redis error

### DISCUSSION THREAD

**Discussion by adrianomic** [mastodon/mastodon#17507] [General]
```
(called from /.../vendor/bundle/ruby/2.7.0/gems/stoplight-2.2.1/lib/stoplight/data_store/redis.rb:28:in `get_all'}
Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.
redis.multi do
  redis.get("key")
end
should be replaced by
redis.multi do |pipeline|
  pipeline.get("key")
end
```

I keeps getting above error message when run test
How to fix that issue?

**Comment by Gargron** [comment]
This deprecation warning is from the redis gem version that is not yet merged into our repository. If you want our support make sure you are not changing lockfiles yourself.

**Reply by adrianomic** [reply]
My redis version is still 4.5 which is same to repo

Anything wrong here?
If you want I will attach gemfile.lock

**Reply by adrianomic** [reply]
I fixed it by downgrade redis in Gemfile.lock
